mkdocs_bootswatch/flatly/css/base.css

```diff
@@ -26,7 +26,23 @@ div.source-links {
 }
 
 div.col-md-9 img {
-  max-width: 100%;
+    max-width: 100%;
+}
+
+code {
+    padding: 1px 3px;
+    background: #ecf0f1;
+    border: solid 1px #ccc;
+    color: #7b8a8b;
```

Comment:
The text colour here isn't great on that background. It doesn't stand out very much, I think the background should be lighter or the foreground darker.